docs/platforms/javascript/common/tracing/instrumentation/index.mdx

```diff
@@ -102,8 +102,33 @@ Sometimes, you do not want the span to be ended automatically when the callback
 
 To add spans that aren't active, you can create independent spans. This is useful when you have work that is grouped together under a single parent span, but is independent from the currently active span. However, in most cases you'll want to create and use the [startSpan](#starting-an-active-span-startspan) API from above.
 
+
 <PlatformContent includePath="performance/start-inactive-span" />
 
+<PlatformCategorySection supported={['browser']}>
+
+  In some cases, you may need to start an inactive span and make it the active span.
+  In contrast to [startSpanManual](#starting-an-active-span-with-manual-end-startspanmanual), the period for which 
+  the span is active is not bound to the callback. Instead, it will remain active until you manually end the span.
```

Comment:
~I am not sure if this is correct. `startSpanManual` also needs to be manually ended, right? Or do I get this sentence wrong 🤔~

After reading it some times again I think you mean that `startSpanManual` is only an active span during the callback, even though it can be ended after the callback. Maybe we can formulate that a bit differently.

"Even though a span can be ended outside the callback of `startSpanManual`, the span continues as inactive outside of the callback. With `setActiveSpan` a span can be set active until you manually end the span."